kobo/apps/accounts/mfa/tests/test_api.py

```diff
@@ -67,3 +73,36 @@ def test_mfa_activation_always_creates_new_secret(self):
             )
             assert first_secret != second_secret
             assert first_response.json() != second_response.json()
+        mfa_availability.delete()
+
+    @override_config(MFA_ENABLED=True)
+    def test_mfa_disabled(self):
```

Comment:
I think we should find a better name for this test. Having `@override_config(MFA_ENABLED=True)` just above `def test_mfa_disabled(self):` is confusing. 
What about something like `test_mfa_whitelisting`

Reply:
Ok